File: src/main/java/br/com/jiratorio/domain/request/CreateBoardRequest.java
package br.com.jiratorio.domain.request;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;
import lombok.Data;

@Data
public class CreateBoardRequest {

    @NotBlank
    private String name;

    @NotNull
    private Long externalId;

}

File: src/main/java/br/com/jiratorio/domain/request/CreateBoardRequest.kt
package br.com.jiratorio.domain.request

import javax.validation.constraints.NotBlank
import javax.validation.constraints.NotNull

data class CreateBoardRequest(
    @field:NotBlank
    var name: String? = null,

    @field:NotNull
    var externalId: Long? = null
)
